Updated prior posterior plot

Jillderon opened the pull request:
X-axis effect size prior posterior plot added

Jillderon committed:
Updated prior posterior plot

X-axis effect size added

Jillderon committed:
Added density table

Added a table with the densities over the prior and posterior in the equivalence and non-equivalence interval

Jillderon committed:
Update grammar error in helpful

Jillderon committed:
Changed helpfiles

Jillderon committed:
Merge branch 'development' of https://github.com/Jillderon/jasp-desktop into development

Jillderon commented:
Would anyone want to look at this so that it can be merged?

vandenman commented:
sorry, I'll take a look later today.

vandenman commented on Resources/Equivalence T-Tests/qml/EquivalenceBayesianIndependentSamplesTTest.qml:
```diff
@@ -16,7 +16,7 @@
 //
 
-import QtQuick			2.12
+import QtQuick			2.8
```
Any particular reason to downgrade QtQuick?

vandenman commented on JASP-Engine/JASP/R/equivalencebayesianpairedsamplesttest.R:
```diff
@@ -243,3 +250,72 @@ EquivalenceBayesianPairedSamplesTTest <- function(jaspResults, dataset, options)
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "separator",   title = " ",                         type = "separator")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "variable2",   title = " ",                         type = "string")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "section",     title = "Section",                   type = "string")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "density",     title = "Density",                   type = "number")
```
```suggestion
  equivalenceDensityPairedTTestTable$addColumnInfo(name = "section",     title = gettext("Section"),                   type = "string")
  equivalenceDensityPairedTTestTable$addColumnInfo(name = "density",     title = gettext("Density"),                   type = "number")
```
add `gettext`?

vandenman commented on JASP-Engine/JASP/R/equivalencebayesianpairedsamplesttest.R:
```diff
@@ -243,3 +250,72 @@ EquivalenceBayesianPairedSamplesTTest <- function(jaspResults, dataset, options)
+      equivalenceDensityPairedTTestTable$addRows(list(variable1     = pair[[1L]], 
+                                                      separator     = "-",
+                                                      variable2     = pair[[2L]],
+                                                      section       = "p(\U003B4 \U02208 I | H1)",
```
So maybe we should think of how to translate this... perhaps something like:

```r
gettextf("p(%1$s I | H1, y)", "\U003B4 \U02208")
```
?
On the other hand that might be a bit unnecessary for this release (it will be the same in Dutch).

vandenman commented on JASP-Engine/JASP/R/equivalencebayesianonesamplettest.R:
```diff
@@ -215,3 +223,63 @@ EquivalenceBayesianOneSampleTTest <- function(jaspResults, dataset, options) {
+                                     "informativeNormalMean", "informativeNormalStd", "informativeTLocation", 
+                                     "informativeTScale", "informativeTDf"))
+  
+  equivalenceDensityTable$addColumnInfo(name = "variable",         title = "Variable",        type = "string", combine = TRUE)
+  equivalenceDensityTable$addColumnInfo(name = "section",          title = "Section",         type = "string")
+  equivalenceDensityTable$addColumnInfo(name = "density",          title = "Density",         type = "number")
```
```suggestion
  equivalenceDensityTable$addColumnInfo(name = "variable",         title = gettext("Variable"),        type = "string", combine = TRUE)
  equivalenceDensityTable$addColumnInfo(name = "section",          title = gettext("Section"),         type = "string")
  equivalenceDensityTable$addColumnInfo(name = "density",          title = gettext("Density"),         type = "number")
```

vandenman commented on JASP-Engine/JASP/R/equivalencebayesianindependentsamplesttest.R:
```diff
@@ -264,3 +271,63 @@ EquivalenceBayesianIndependentSamplesTTest <- function(jaspResults, dataset, opt
+                                  "informativeNormalMean", "informativeNormalStd", "informativeTLocation", 
+                                  "informativeTScale", "informativeTDf"))
+  
+  equivalenceDensityTable$addColumnInfo(name = "variable",         title = "Variable",        type = "string", combine = TRUE)
+  equivalenceDensityTable$addColumnInfo(name = "section",          title = "Section",         type = "string")
+  equivalenceDensityTable$addColumnInfo(name = "density",             title = "Density",      type = "number")
```
```suggestion
  equivalenceDensityTable$addColumnInfo(name = "variable",         title = gettext("Variable"),        type = "string", combine = TRUE)
  equivalenceDensityTable$addColumnInfo(name = "section",          title = gettext("Section"),         type = "string")
  equivalenceDensityTable$addColumnInfo(name = "density",             title = gettext("Density"),      type = "number")
```

vandenman reviewed (changes requested):
The changes are a bit too copy-paste for my flavor, but that's probably something to look at after this release.

I found one bug:

![image](https://user-images.githubusercontent.com/21319932/78242342-20350d00-74e2-11ea-9622-5dc66c8eb861.png)

To reproduce:
1. Open analysis
2. Enter `contNormal`.
3. Tick any option. It looks like the object is only made when it doesn't exist in the state, but when it exists in the state it is never extracted.

Jillderon committed:
Update JASP-Engine/JASP/R/equivalencebayesianpairedsamplesttest.R

Co-Authored-By: Don van den Bergh <donvdbergh@hotmail.com>

Jillderon committed:
Update JASP-Engine/JASP/R/equivalencebayesianonesamplettest.R

Co-Authored-By: Don van den Bergh <donvdbergh@hotmail.com>

Jillderon committed:
Update JASP-Engine/JASP/R/equivalencebayesianindependentsamplesttest.R

Co-Authored-By: Don van den Bergh <donvdbergh@hotmail.com>

Jillderon committed:
Inferiority testing

Incorporated Don's feedback and added inferiority testing to the Bayesian equivalence testing

Jillderon committed:
Merge branch 'development' of https://github.com/Jillderon/jasp-desktop into development

Jillderon committed:
change table name

From density to mass

Jillderon committed:
helpfiles changed

Jillderon committed:
gettext()

Jillderon committed:
Merge branch 'development' into development

Jillderon committed:
dependencies fixed

Added new dependencies

Jillderon committed:
Merge branch 'development' of https://github.com/Jillderon/jasp-desktop into development

vandenman reviewed (commented):
So the changes look fine to me. Just one thing that stands out while testing. To reproduce:

1. Open the one sample.
2. Add `contNormal`.
3. Tick `Prior and posterior` under `Plots`.
4. Change the equivalence region to `from -Inf to 0.5`.
5. The plot disappears.

For all three analyses, the plots disappear when you change the equivalence region to `from -Inf to 0.5`. Is this intended?

Jillderon committed:
Incorporated feedback don

Posterior and prior plot were not showing when the Bayes Factor was infinity. Now a plot displays with a warning.

vandenman commented:
@boutinb The changes look good to me, if travis passes then this can be merged.
